README.md

```diff
@@ -61,6 +68,35 @@ Learn more:
 
 If you find CSV files that it can't parse, send me examples or pull requests!
 
+## Unattended mode
+
+You can run reckon in a non-interactive mode.
+To guess the accounts reckon can use an existing ledger file or a token file with keywords.
+
+`reckon --unattended -l 2010.dat -f bank.csv -o ledger.dat`
+
+`reckon --unattended --account-tokens tokens.yaml -f bank.csv -o ledger.dat`
+
+Here's an example of `tokens.yaml`:
+
+```
+Income:
+  Salary:
+    - 'LÖN'
+    - 'Salary'
+Expenses:
+  Bank:
+    - 'Comission'
+    - 'MasterCard'
+  Rent:
+    - '0011223344' # Landlord bank number
+'[Internal:Transfer]': # Virtual account
+  - '4433221100' # Your own account number
+```
+
+If reckon can not guess the accounts it will use `Income:Unknown` or `Expenses:Unknown` names.
+You can override them with `--default_outof_account` and `--default_into_account` options.
```

Comment:
For consistency, maybe it should be `--default-outof-account` and `--default-into-account` (with hyphens)?